test/local/metrics/events.js

```diff
@@ -220,6 +220,212 @@ describe('metrics/events', () => {
       })
   })
 
+  it('.emit with flow event and no session token', () => {
+    const time = Date.now()
+    sinon.stub(Date, 'now', () => time)
+    const metricsContext = mocks.mockMetricsContext()
+    const request = {
```

Comment:
Looks like the `const request` object is very similar between these test cases. Could have a helper function to build out the differences.